ClientApp/src/Pages/Home/Home.tsx

```diff
@@ -1,7 +1,19 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect, useState } from 'react';
 import './Home.scss';
+import { GetSightings } from './GetSightings';
+import {getRecentSightings} from '../../Api/apiClient';
+import {RecentSightingResponse, RecentSightingResponseList} from '../../Api/apiClient';
 
 function Home() {
+
+  const [myData, setMyData] = useState<RecentSightingResponseList | null>(null);
```

Comment:
could we call this recentSightings or something similar instead of myData please

Reply:
I don't think this has been renamed yet :) have you pushed your changes?